docs/getting-started.md

```diff
@@ -29,10 +29,10 @@ Pentagon is “batteries included”- not only does one get a network with a clu
 
 ## Quick Start
 ### Create a AWS Pentagon Project
-* `pentagon start-project <project-name> --aws-access-key <aws-access-key> --aws-secret-key <aws-secret-key> --aws-default-region <aws-default-region>`
+* `pentagon start-project <project-name> --aws-access-key <aws-access-key> --aws-secret-key <aws-secret-key> --aws-default-region <aws-default-region>` --dns-zone <your-dns-zone>
```

Comment:
`--dns-zone` is not strictly required. It is only required where the dns-zone doesn't match the project name. Do you think its necessary to document that here?

Reply:
That's up to you.  In my experience, the domain almost never matches the project name.  Since this is a quick-start guide, I felt it would be more likely to get people working rather than having to read more docs to figure out how to troubleshoot an error when they miss that detail.  

It doesn't hurt to specify it if it does match the project name, right?

Reply:
It _should not_ hurt. I agree, it usually doesn't match. Let's leave it in.